seqr/models.py

```diff
@@ -327,7 +330,7 @@ def __unicode__(self):
         return self.individual_id.strip()
 
     def _compute_guid(self):
-        return 'I%06d_%s' % (self.id, _slugify(str(self)))
+        return 'I%07d_%s' % (self.id, _slugify(str(self)))
```

Comment:
why change this?

Reply:
I increased other ones, so figured it wouldn't hurt